src/node_state.rs

```diff
@@ -364,6 +364,15 @@ impl NodeState {
 
         let mut should_try_and_settle = true;
 
+        // For now, reject blob Tx with blobs for unknown contracts.
```

Comment:
the txs could also be refused at api level from the node to avoid spaming of network

Reply:
I don't think we have a map of contracts in the API for now